proto/verifiable-credential/verifiable-credential.proto

```diff
@@ -41,24 +49,102 @@ definition: https://w3c.github.io/vc-data-model/
+  // property MUST be a string value of an [RFC3339] combined date and time string
+  // representing the date and time the credential becomes valid, which could
+  // be a date and time in the future. Note that this value represents the earliest
+  // point in time at which the information associated with the credentialSubject
+  // property becomes valid.
   string issuance_date = 5;
+  // The value of the credentialSubject property is defined as a set of
+  // objects that contain one or more properties that are each related
+  // to a subject of the verifiable credential.
   oneof credential_subject {
+    // The value of user_cred represents a privacy respecting verifiable
+    // credential. This is used when adding sensitive information about
+    // a credential subject. It allows the credential subject to create
+    // and validate proofs about what is contained in a credential without
+    // revealing the values contained in the credential otherwise known as
+    // selective disclosure.
     UserCredentialSubject user_cred = 6;
+
+    // The value of license_cred represents a license issued by a regulatory
+    // body. The license can be used to define authorized actions by the 
+    // credential subject
+    LicenseCredential license_cred = 7;
   };
-  Proof proof = 7;
+
+  // One or more cryptographic proofs that can be used to detect tampering
+  // and verify the authorship of a credential or presentation. The specific
+  // method used for an embedded proof MUST be included using the type property.
+  Proof proof = 8;
 }
 
-message UserCredentialSubject {
+// UserCredentialSubject represents a privacy respecting
+// credential_subject of a verifiable credential. This
+// is used as an on chain verifiable credential.
+message UserCredentialSubject { 
   string id = 1;
   string root = 2;
   bool is_verified = 3;
 }
 
+// The LicenseCredential message makes reference to the classes of crypto assets
+// described in MiCA, but the license could easily be adopted as proof of
+// authority to issue various types of crypto or virtual assets. The LicenseCredential
+// is used a a credential_subject in a verifiable credential.
+message LicenseCredential {
+  // The value of id represents the ID of the credential_subject
+  string id = 1;
+
+  // The license type is defined by the MICA regulation. This will
+  // be used to identify different asset classes of tokens being issuedi
+  // by the credential_subject.
+  string license_type = 2;
+
+  // The country field represents the country the credential was issued in.
+  string country = 3;
+
+  // The authority field represents a licensing authority that issued the LicenseCredential
+  string authority = 4;
+
+  // The denom field of this message represents a token name
+  // this will be used to validate issuer denoms.
+  string denom = 5;
+
+  // The circulation_limit represents the amount of a token 
+  // that can be minted by a credential_subject.
+  uint64 circulation_limit = 6;
```

Comment:
should we use the Coin type for amounts? see 
https://github.com/cosmos/cosmos-sdk/blob/3f8596c1955e40ef30e4abcd06f2237d132db3a9/proto/cosmos/bank/v1beta1/tx.proto#L26

 ```suggestion
  cosmos.base.v1beta1.Coin circulation_limit = 6  [(gogoproto.nullable) = false];
```